core/src/main/java/nu/yona/server/analysis/service/ActivityDTO.java

```diff
@@ -18,22 +16,25 @@
 @JsonRootName("activity")
 public class ActivityDTO
 {
-	private Date startTime;
-	private Date endTime;
+	private ZonedDateTime startTime;
+	private ZonedDateTime endTime;
 
 	@JsonCreator
-	public ActivityDTO(@JsonProperty("startTime") Date startTime, @JsonProperty("endTime") Date endTime)
+	public ActivityDTO(@JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ") @JsonProperty("startTime") ZonedDateTime startTime,
```

Comment:
I take it that you added this comment before seeing that I already centralized the pattern, right?